sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegenExec.scala

```diff
@@ -479,8 +482,7 @@ case class CollapseCodegenStages(conf: SQLConf) extends Rule[SparkPlan] {
   }
 }
 
-
-case class WholeStageCodegenRDD(@transient sc: SparkContext, var source: CodeAndComment,
+private[spark] class WholeStageCodegenRDD(@transient sc: SparkContext, var source: CodeAndComment,
```

Comment:
I guess I can revert that, as I was trying to extend this RDD , this was the fallout of that. Now that I am not I can make it as a case class again.